src/azure-cli-core/azure/cli/core/commands/parameters.py

```diff
@@ -248,6 +248,8 @@ def _type(value):
 
 name_type = CLIArgumentType(options_list=['--name', '-n'], help='the primary resource name')
 
+edge_zone_type = CLIArgumentType(help='The name of edge zone.', is_preview=True)
```

Comment:
Must it be in core?

Reply:
There are multiple operations in different modules to use the argument, so we define it here. If not in core, each command has to define by themselves.